src/aap_eda/services/activation/manager.py

```diff
@@ -633,6 +633,9 @@ def _error_instance(self, msg: tp.Optional[str] = None):
 
     def _stop_instance(self):
         """Stop the latest activation instance."""
+        log_handler = DBLogger(self.latest_instance.id)
+        log_handler.write("ACTIVATION BEING STOPPED AT USER REQUEST", True)
+
```

Comment:
I just have merged changes precisely for this. :) You will have conflicts, better to remove this.

Reply:
👌